packages/docs/operations-manual/overview.md

```diff
@@ -0,0 +1,13 @@
+# Validator Guide Overview
```

Comment:
Just a comment, but I find the directory name `operations-manual` to be a little confusing.

Reply:
agreed, I added this as a point to address in the docs upgrade proposal

Reply:
we can safely rename it `validator-guide` without affecting any URLs since this is what the section heading slug will be